snuba/query/processors/type_converters/hexint_column_processor.py

```diff
@@ -0,0 +1,22 @@
+from snuba.query.expressions import Column, Expression, FunctionCall, Literal
+from snuba.query.processors.type_converters import BaseTypeConverter, ColumnTypeError
+
+
+class HexIntColumnProcessor(BaseTypeConverter):
```

Comment:
Shouldn't there be another class extending BaseTypeCOnverter for UUIDs ?

Reply:
I should have mentioned this but I'm planning to tackle it as a follow p. I didn't want to ship a big change at once since UUID are widely used.

Reply:
ok. that's fine to me